analyzers/its/regression-test.ps1

```diff
@@ -226,6 +226,21 @@ function Show-DiffResults() {
     } -errorMessage $errorMsg
 }
 
+function Invoke-JsonParser()
+{
+    $jsonParser = Join-Path $PSScriptRoot "..\packaging\binaries\ITs.JsonParser\ITs.JsonParser.exe"
+    $arguments = @(" ")
```

Comment:
All over the (new) code
```suggestion
    $Arguments = @(" ")
```

Reply:
What's the difference?
The old code already uses camelCase, and according to [this page](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_variables?view=powershell-7.4) they use both camel and pascal case 🤔

Reply:
I did the change, but now it looks a bit incosistent

Reply:
The annoyingCase is a solution to a problem that doesn't exist in case-insensitive languages.

It is also strongly recommended by MS: https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.4#use-pascal-case-for-parameter-names

Reply:
Isn't this only about parameters?
E.g. `ruleId` and `project` in our case, or for functions.